docker/Dockerfile.dev

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu
+
+MAINTAINER perryism
+ 
+RUN apt-get update && \
+    apt-get -y install \
+	wget \
+	zip \
+	python-pip \
+	git \
+    && rm -rf /var/lib/apt/lists/* 
+
+ARG VERSION=0.8.8
+
+RUN wget https://github.com/airbnb/knowledge-repo/archive/v$VERSION.zip && \
+    unzip v$VERSION.zip -d /app
+
+WORKDIR /app/knowledge-repo-$VERSION
+
+COPY . /app/knowledge-repo-$VERSION
+
+RUN pip install -r requirements.txt
+
+COPY entrypoint.sh /app/knowledge-repo-$VERSION
```

Comment:
When copying shell scripts you should also set sufficient permissions inside the container. Otherwise the permissions of your host system are used as default.

You could add `RUN chmod +x /app/knowledge-repo-$VERSION/entrypoint.sh` or instead specify a user that is running the process.